### DISCUSSION TITLE
jsdelivr being blocked by company firewall. Can we have a way to disable it and just use codesandbox domain?

### DISCUSSION THREAD

**Discussion by pastinepolenta** [codesandbox/codesandbox-client#6952] [Q&A]
As jsdelivr is being blocked by my company's firewall can we have a way to disable it and just use codesandbox domain? Or maybe a way to configure with package provider to use?

**Comment by DeMoorJasper** [comment]
We have a fallback to unpkg if jsdelivr is offline/blocked. But that is probably also blocked? For a lot of modules we're already using our own CDN, I guess we could go further and also proxy jsdelivr/unpkg will ask internally what we can do here
